.vscode/launch.json

```diff
@@ -0,0 +1,18 @@
+{
```

Comment:
why `.vscode` is not added to `.gitignore`?

Reply:
@slavaleleka, or, better, to the developer's personal global gitignore :grin:.

<https://stackoverflow.com/a/7335487/1892060>